installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -142,8 +142,6 @@ spec:
                   key: keptn-api-token
             - name: CLI_DOWNLOAD_LINK
               value: "{{ .Values.bridge.cliDownloadLink | default (print "https://github.com/keptn/keptn/releases/tag/" .Chart.AppVersion) }}"
-            - name: INTEGRATIONS_PAGE_LINK
-              value: "{{ .Values.bridge.integrationsPageLink | default (print "https://get.keptn.sh/integrations.html") }}"
```

Comment:
There are more references to the `INTEGRATIONS_PAGE_LINK`, please remove them as well.

See https://github.com/keptn/keptn/blob/0aaededa6057f09e28dd4f6b0b90e2f9cb3dcec5/bridge/deploy/service.yaml#L43-L44 and https://github.com/keptn/keptn/blob/0589b262fa1844097ee499c8cc153fdefca4f614/installer/manifests/keptn/charts/control-plane/templates/core.yaml#L145-L146 and https://github.com/keptn/keptn/blob/aa4bcf0eda42faa8d108bafee7a711ad60f73ad0/bridge/server/app.ts#L29 and https://github.com/keptn/keptn/blob/aa4bcf0eda42faa8d108bafee7a711ad60f73ad0/bridge/server/app.ts#L82-L85 and https://github.com/keptn/keptn/blob/aa4bcf0eda42faa8d108bafee7a711ad60f73ad0/bridge/server/api/index.ts#L23 and https://github.com/keptn/keptn/blob/aa4bcf0eda42faa8d108bafee7a711ad60f73ad0/bridge/server/api/index.ts#L33 and https://github.com/keptn/keptn/blob/aa4bcf0eda42faa8d108bafee7a711ad60f73ad0/bridge/server/api/index.ts#L75-L85

Reply:
all the changes were made